docs/vectorizer.md

```diff
@@ -140,13 +152,15 @@ The view includes all columns from the blog table, plus the following additional
 To find the closest embeddings to a query, use this canonical SQL query:
 
 ```sql
-SELECT 
-   chunk,
-   embedding <=> <query embedding> as distance
+SELECT chunk,
+   embedding <=> ai.openai_embed('text-embedding-3-small', 'my query string', _dimensions=>768)  as distance
```

Comment:
Is the `_dimensions=>768` parameter supposed to be private or why does it lead with underscore? without it, the dimensions are too big and I got an error.

Reply:
Good point we may need to remove the underscore @jgpruitt

Reply:
all the parameters in versions 0.1.0, 0.2.0, and 0.3.0 use leading underscores. it was just a habit I have, because leading underscores prevent name clashes with keywords, reserved words, and object/column names.